### PR TITLE
Cleanup of anon routines.

### PR DESCRIPTION
**Pull request by mvsjes2**
Allow anon routines to be protected so subclasses can call them.

### CODE REVIEW

File: lib/Sub/Override.pm [hunk 1]
```diff
@@ -187,6 +121,62 @@ sub DESTROY {
     }
 }
 
+sub _normalize_sub_name {
```
Comment (Ovid) [inline lib/Sub/Override.pm:124]: A tiny nit (my fault) that's bugged me for a long time. This should probably be renamed something like `_get_fully_qualified_sub_name` or something like that.

**Review by Ovid — APPROVED** [review]
Tiny comment about a small mistake I made a long time ago, but not a blocker 😃 Approved!

And thank you for maintaining this. I really appreciate it.

File: lib/Sub/Override.pm (hunk 1 above)
Comment (mvsjes2) [reply lib/Sub/Override.pm]: Good idea. I noticed my wrap routine has a mis-named variable. I may take a look at the variable names in general and fix anything else I find, sometime.

**Comment by mvsjes2** [comment]
I've 'paused' version 0.12 and see it on cpan now.

**Pull request merged**